src/Storefront/Resources/views/storefront/component/address/address-personal-vat-id.html.twig

```diff
@@ -1,5 +1,5 @@
 {% block component_account_register_personal_vat_id_field %}
-    {% set vatIdValue = activeRoute == 'frontend.account.profile.page' or (activeRoute == 'frontend.account.addressbook' and context.customer.guest) ? vatIds[0] : vatIds.get(0) %}
+    {% set vatIdValue = activeRoute == 'frontend.account.profile.page' or (activeRoute == 'frontend.account.addressmanager.get' and context.customer.guest) ? vatIds[0] : vatIds.get(0) %}
```

Comment:
@FlorianKe is this the proper replacement here? the old route was still used here, and this was mentioned as replacement

Reply:
The address editor plugin is also still referenced in the `main.js`

```JavaScript
PluginManager.register('AddressEditor', () => import('src/plugin/address-editor/address-editor.plugin'), '[data-address-editor]');
```

I will quickly remove that.